### PR TITLE
update Faster and Mask RCNN input shapes

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

**Comment by kchechil** [comment]
@eaidova could you please create a branch in Gitlab referring to the new OMZ as submodule? I'll run quantization jobs to check it. Thank you.

**Comment by eaidova** [comment]
@kchechil, only if I replace original OMZ with my fork right now... I does not have permissions to merge these changes, sorry
If it is OK for you, then I do it

**Comment by kchechil** [comment]
@eaidova I guess that's fine if you do it on branch - this is only for the test. However, when submitting the real PR with OMZ update, we will need to have it merged + submodule should refer to AC/OMZ from dev branch.

File: models/public/faster_rcnn_inception_v2_coco/faster_rcnn_inception_v2_coco.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Use Case and High-Level Description
 
-Faster R-CNN with Inception v2. Used for object detection. For details, see the [paper](https://arxiv.org/pdf/1801.04381.pdf).
+Faster R-CNN with Inception v2. Used for object detection. For details, see the [paper](https://arxiv.org/pdf/1506.01497v3.pdf).
```
Comment (IRDonch) [inline models/public/faster_rcnn_inception_v2_coco/faster_rcnn_inception_v2_coco.md]: Shouldn't a similar change be made to `faster_rcnn_inception_resnet_v2_atrous_coco`, as well?

**Comment by IRDonch** [comment]
Could you explain this change? Were the original shapes just wrong to begin with?

**Comment by kchechil** [comment]
@IRDonch the models were trained for the input resolution 600x1024 (faster) and 800x1365 (mask_rcnn). Because of that, accuracy metrics obtained for the original resolution was lower. So it was decided to update the input resolution to get adequate accuracy metrics.

File: models/public/faster_rcnn_inception_v2_coco/faster_rcnn_inception_v2_coco.md (hunk 1 above)
Comment (eaidova) [reply models/public/faster_rcnn_inception_v2_coco/faster_rcnn_inception_v2_coco.md]: updated, thanks

**Comment by eaidova** [comment]
@vladimir-dudnik @IRDonch  could you please merge this PR? It is blocked submodules update in calibration.

**Pull request merged**